roles/openshift_manageiq/tasks/main.yaml

```diff
@@ -59,6 +59,16 @@
   failed_when: "'already exists' not in osmiq_perm_task.stderr and osmiq_perm_task.rc != 0"
   changed_when: osmiq_perm_task.rc == 0
 
+- name: Configure 3_2 role/user permissions
+  command: >
+    {{ openshift.common.admin_binary }} {{item}}
+    --config={{manage_iq_tmp_conf}}
+  with_items: "{{manage_iq_openshift_3_2_tasks}}"
+  register: osmiq_perm_3_2_task
+  failed_when: osmiq_perm_3_2_task.rc != 0
+  changed_when: osmiq_perm_3_2_task.rc == 0
```

Comment:
> Yes I am. Question is what to use for changed_when

I don't know what change_when is.  You can get the binding using `oc get clusterpolicybinding` (no name arg).  It will list them all (should only be one) and you can search for your item.  It will be painful.

We don't guarantee rolebinding names, but `oc get clusterrolebinding/<find your name>` would probably be pretty stable.